### ISSUE TITLE
Lua error in gmod

### ISSUE THREAD

**Issue by zmoote** [viral32111/black-hole-cache#2]
Hi, I went to load into a map today and BHC was giving this error in the 'problems' section:

[Black Hole Cache] lua/autorun/blackholecache.lua:26: attempt to compare number with nil
    1. onsuccess - lua/autorun/blackholecache.lua:26
        2. unknown - lua/includes/modules/http.lua:29

**Comment by viral32111** [comment]
You're probably using the workshop version, which is out of date. The [code responsible for this error has been commented out on this repository for several months](https://github.com/viral32111/black-hole-cache/blob/0e49344a08cdf641f19c04af69499499a948c2a1/lua/autorun/blackholecache.lua#L23-L40).

A few months ago I started making a rewrite of this addon on this repository which slightly broke [the current workshop version](https://github.com/viral32111/black-hole-cache/tree/16db3b6224cfd051c9138bc457673e31bbb2ae1b), but never got it finished as I abandoned coding Garry's Mod addons not too long ago. I might come back to it in the future, but as for right now I'm not going to be doing anything to fix this.

The addon should still work as normal, the only code that's broken is the version checker, so nothing to worry about since there won't be any new versions for the forseeable future. If this error is really bugging you though, I've attached a fixed version of the addon below with the version checker code commented out which can be extracted and dropped into your game installation's `addons` folder, just make sure you unsubscribe from the workshop version so there aren't any conflicts.

[Download `black-hole-cache-16db3b6.zip`.](https://github.com/viral32111/black-hole-cache/files/6598742/black-hole-cache-16db3b6.zip)
